overlord/assertstate/assertstate_test.go

```diff
@@ -962,6 +965,7 @@ func (s *assertMgrSuite) TestRefreshSnapDeclarationsNop(c *C) {
 
 	err := assertstate.RefreshSnapDeclarations(s.state, 0)
 	c.Assert(err, IsNil)
+	c.Assert(s.fakeStore.(*fakeStore).opts.IsAutoRefresh, Equals, false)
```

Comment:
Thanks for adding this! 

As another nitpick, I think `c.Check()` is slightly better here and below but we have no hard rules for when to use Assert and Check, it's a bit mixed. Below there is a bunch more that the test could check even when the IsAutoRefresh is not set. In general the rule of thumb is that Assert() is useful if it does not make sense to test anything beyond the this point and Check when one detail about the test is not correct but it may make sense to continue. With assert() one has to fix issues one-by-one, i.e. run test, fix one assert, run again, fix next assert etc. With Check() multiple issues are visible in the same test run.